workflow/react-project/src/components/generic/Modal/style.less

```diff
@@ -0,0 +1,74 @@
+@import "../../../styles/fonts.less";
+@import "../../../styles/colors.less";
+@import "../../../styles/mixins.less";
+
+.dark-background {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: 1;
+  background-color: rgb(0 0 0 / 38%);
+}
+
+.modal-wrapper {
+  width: 80vw;
+  height: 60vh;
+  z-index: 3;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  @media (min-width: 500px) {
+    width: min(600px, 80vw);
+    height: min(700px, 60vh);
+  }
+}
+
+.modal-header {
+  .flex-between-vertical-centered;
+
+  width: 100%;
+  padding: 10px;
+  border-radius: 10px 10px 0 0;
+  background: @green-color;
+
+  @media (min-width: 768px) {
```

Comment:
some of the breakpoints you can move to separate file (in `styles` folder, for ex. `constants.less`)
got ex: `@MIN_DESKTOP_WIDTH: 768px`